src/Playwright/Core/Page.cs

```diff
@@ -48,6 +48,8 @@ internal class Page : ChannelOwner, IPage
     private List<RouteHandler> _routes = new();
     private Video _video;
     private string _closeReason;
+    internal bool _closeWasCalled = false;
```

Comment:
This could by an internal property instead of exposing the variable :)

Reply:
Done.